src/index.html

```diff
@@ -25,6 +25,9 @@
         <lisk-button-send amount=100 recipient="15015136092749848942L" button-title="Send Me LSK for food"></lisk-button-send>
         <lisk-button-vote votes="goodPerson1,goodPerson2,goodPerson3" unvotes="badPerson1,badPerson2,badPerson3"></lisk-button-vote>
         <lisk-button-vote votes="goodPerson1,goodPerson2,goodPerson3" button-title="Vote for ME"></lisk-button-vote>
+        <lisk-button-sign type="hub" message="sign with hub" button-title="Sign by using Lisk Hub!"></lisk-button-sign>
+        <lisk-button-sign message="sign with hub 2"></lisk-button-sign>
```

Comment:
What is the 2?

Reply:
It helps during real test to have a visual feedback that both are working

Reply:
ah lol true, that's the index.html :)